src/main/java/com/blamejared/brackets/util/IMaterialDefinition.java

```diff
@@ -0,0 +1,20 @@
+package com.blamejared.brackets.util;
+
+import stanhebben.zenscript.annotations.*;
+
+/**
+ * Created by Jared on 6/16/2016.
+ */
+@ZenClass("modtweaker.material.IMaterialDefinition")
+public interface IMaterialDefinition {
+
+    @ZenOperator(OperatorType.MUL)
```

Comment:
I would say make "asMaterial" an `ZenGetter("material")`, just so people can get the material from the def